### PR TITLE
Document guidance for community to test pre-releases and releases

### LINKED ISSUE #1809: Document guidance for community to test pre-releases and releases
As a community member, I would like specific guidance on how to test the pieces of an OSCAL release so that I can provide feedback to the OSCAL development team.

## Acceptance Criteria

- [x] A new page on the OSCAL NIST pages site
- [ ] Guidance on how to:
    - [ ] Validate content against a release's schemas
    - [ ] Perform XML transformations in a release

### PR DESCRIPTION
**Pull request by nikitawootten-nist**
# Committer Notes

Closes #1809

Also in this PR:
- Fix stray newline in PR template

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Do all automated CI/CD checks pass?
- ~Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]~

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~~Have you written new tests for your core changes, as applicable?~~
- ~~Have you included examples of how to use your new feature(s)?~~
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: docs/content/learn/tutorials/general/releases.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+The OSCAL team encourages community members to submit feedback and ask questions as [GitHub issues](https://github.com/usnistgov/OSCAL/issues) or to the [release's accompanying GitHub discussion](https://github.com/usnistgov/OSCAL/discussions/categories/releases).
+
+## Testing a new release
+
+All releases are published to [OSCAL GitHub repository](https://github.com/usnistgov/OSCAL) in the [releases section](https://github.com/usnistgov/OSCAL/releases).
+
+Releases are [tagged in Git](https://git-scm.com/book/en/v2/Git-Basics-Tagging) automatically with a "v" prefix.
+For example, a release for version 1.0.0 would be tagged `v1.0.0`.
+
+To test a specific release, [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and checkout the release's associated tag.
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:37]: Minor, non-blocking: add some whitespace like other paragraphs?

```suggestion
To test a specific release, [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and checkout the release's associated tag.

```

File: docs/content/learn/tutorials/general/releases.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+cd path/to/OSCAL
+# Fetch the newest changes to the repository from GitHub
+git fetch --all
+# Checkout the "v1.0.5" tag
+git checkout v1.0.5
+```
+
+### How do I validate existing content against a new release?
+
+In order to to maintain a backwards-compatibility guarantee, the OSCAL team must rely on the community to validate their content against new releases and most importantly pre-releases.
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:61]: Minor whitespace

```suggestion
In order to to maintain a backwards-compatibility guarantee, the OSCAL team must rely on the community to validate their content against new releases and most importantly pre-releases.

```

File: docs/content/learn/tutorials/general/releases.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+For more details on validation, see the [validation concept page](/concepts/validation/).
+
+OSCAL schemas are generated as part of a release using the source [Metaschema definitions](/concepts/layer/overview/#modeling-approach).
+
+Additional constraints are imposed on the OSCAL models that are not representable within the JSON and XML schemas.
+These constraints can be validated using downstream tools, such as the [OSCAL CLI](/tools/#nists-oscal-application-frameworks-tools-and-libraries).
+Accompanying pre-releases for these tools may be created as appropriate.
+For more details see ["How do I test downstream tooling?"](#how-do-i-test-downstream-oscal-tooling).
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:69]: Minor whitespace changes

```suggestion
Additional constraints are imposed on the OSCAL models that are not representable within the JSON and XML schemas.

These constraints can be validated using downstream tools, such as the [OSCAL CLI](/tools/#nists-oscal-application-frameworks-tools-and-libraries).

Accompanying pre-releases for these tools may be created as appropriate.

For more details see ["How do I test downstream tooling?"](#how-do-i-test-downstream-oscal-tooling).
```

File: docs/content/learn/tutorials/general/releases.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+#### Validating XML OSCAL content
+
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:72]: Important to call out here:

```suggestion

{{<callout>}}We appreciate the community using [example content from NIST](https://github.com/usnistgov/oscal-content/blob/main/examples/) and others, but we strongly encourage the community to test validation against real-world, production-grade data.{{</callout>}}
```

File: docs/content/learn/tutorials/general/releases.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+As an example, a piece of OSCAL content can be validated against a schema using [`libxml2`](https://gitlab.gnome.org/GNOME/libxml2)'s [`xmllint`](https://gnome.pages.gitlab.gnome.org/libxml2/xmllint.html):
+
+```sh
+$ xmllint --noout \
+  --schema OSCAL/xml/schema/oscal_complete_schema.xsd \
+  example_oscal_content.xml
+```
+
+#### Validating JSON and YAML OSCAL content
+
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:85]: Call out here as well:

```suggestion

{{<callout>}}We appreciate the community using [example content from NIST](https://github.com/usnistgov/oscal-content/blob/main/examples/) and others, but we strongly encourage the community to test validation against real-world, production-grade data.{{</callout>}}

```

File: docs/content/learn/tutorials/general/releases.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+To use the XML tools from a release, first follow the environment setup instructions located in [`/build/README.md`](https://github.com/usnistgov/OSCAL/tree/main/build) and [`/build/ci-cd/README.md`](https://github.com/usnistgov/OSCAL/tree/main/build/ci-cd).
+The [Docker](https://www.docker.com/) container definition (located in [`/build/Dockerfile`](https://github.com/usnistgov/OSCAL/blob/main/build/Dockerfile)) provides a simple way to replicate the build environment reproducibly.
+
+#### Conversion
+
+Detailed instructions for performing OSCAL content conversion from XML to JSON are located at [`/json/README.md`](https://github.com/usnistgov/OSCAL/tree/main/json).
+Detailed instructions for performing OSCAL content conversion from JSON to XML are located at [`/xml/README.md`](https://github.com/usnistgov/OSCAL/tree/main/xml).
+
+#### Profile Resolution
+
```
Comment (aj-stein-nist) [inline docs/content/learn/tutorials/general/releases.md:113]: Important to call out:

```suggestion

{{<callout>}}We appreciate the community using [example content from NIST](https://github.com/usnistgov/oscal-content/blob/main/examples/) and others, but we strongly encourage the community to test profile resolution against real-world, production-grade data.{{</callout>}}

```

**Review by aj-stein-nist — APPROVED** [review]
Minor changes and some call-outs, but I am happy with these additions. If you are not comfortable with my recommendations, feel to not pull them in.

File: docs/content/learn/tutorials/general/releases.md (hunk 1 above)
Comment (nikitawootten-nist) [reply docs/content/learn/tutorials/general/releases.md:37]: I'm dismissing whitespace related feedback here, I tend to follow the convention of [one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) for markdown documents. This means that a paragraph will have each sentence on its own line.

Comment (aj-stein-nist) [reply docs/content/learn/tutorials/general/releases.md:37]: I'm cool with that, I just have trouble quickly skimming and knowing if, in GitHub, if the newline is just flow wrapping of a phrase and is intentionally a new paragraph without an empty line between paragraphs in such Markdown files. I am fine without, hence I allow you to ignore it. 😄 Thanks again!

**Pull request merged**